source/_components/zoneminder.markdown

```diff
@@ -0,0 +1,31 @@
+---
+layout: page
+title: "ZoneMinder"
+description: "Instructions how to integrate ZoneMinder into Home Assistant."
+date: 2016-10-13 22:57
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: zoneminder.png
+ha_category: Camera
+featured: false
+---
+
+The ZoneMinder component is the main component to integrate all the sensors and switches for controlling your ZoneMinder instance.
+
+```yaml
+# Example configuration.yaml entry
+zoneminder:
+  host: ZM_HOST
+  url: ZM_PATH
+  username: USERNAME
+  password: PASSWORD
+```
+
+Configuration variables:
+- **host** (*Required*): Your ZoneMinder server.
+- **url** (*Optional*): Path to your ZoneMinder install (default: /zm/)
```

Comment:
`url` as a variable seems a bit off since it's generally a full address like `http://home-assistant.io/`